code/game/objects/structures/crates_lockers/closets/secure/militia.dm

```diff
@@ -137,14 +141,16 @@
 	new /obj/item/ammo_magazine/speed_loader_rifle_75(src)
 	new /obj/item/ammo_magazine/speed_loader_rifle_75(src)
 	new /obj/item/ammo_magazine/speed_loader_rifle_75(src)
-	new /obj/item/ammo_magazine/speed_loader_rifle_75(src)
```

Comment:
why are you removing this one?

Reply:
The pouch can only hold 3.

Reply:
you load 1 clip into gun and have 3 in pounch